src/main/java/at/gammastrahlung/monopoly_server/game/Game.java

```diff
@@ -1,19 +1,20 @@
 package at.gammastrahlung.monopoly_server.game;
-
 import at.gammastrahlung.monopoly_server.game.gameboard.Field;
 import at.gammastrahlung.monopoly_server.game.gameboard.GameBoard;
+import at.gammastrahlung.monopoly_server.game.gameboard.Property;
 import com.google.gson.annotations.Expose;
-
-import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.Setter;
-
 import java.security.SecureRandom;
 import java.util.*;
 import java.util.concurrent.ConcurrentHashMap;
 
-@AllArgsConstructor
+@Getter
```

Comment:
There is a reason why the getter and setter annotations are not for the whole class, as some thigs should not be able to be set.
Remove this or change the other annotations